### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by knightXun**
fix jobcontroller typo

### CODE REVIEW

**Comment by knightXun** [comment]
that's careless error

File: pkg/controller/job/jobcontroller.go [hunk 1]
```diff
@@ -540,7 +540,7 @@ func newCondition(conditionType batch.JobConditionType, reason, message string)
 	}
 }
 
-// getStatus returns no of succeeded and failed pods running a job
+// getStatus returns number of succeeded and failed pods in a running job
```
Comment (cblecker) [inline pkg/controller/job/jobcontroller.go]: This doesn't appear to be a typo. This also seems to read like you pass a job to this function, when you're actually passing in pods.

Comment (knightXun) [inline pkg/controller/job/jobcontroller.go]: agree, but there is a syntax error. maybe make greenhand confused.

Comment (janetkuo) [inline pkg/controller/job/jobcontroller.go]: How about: 
```go 
// getStatus returns the number of succeeded and failed pods from a given list of pods 
```

This function actually has nothing to do with jobs, although it's used by job controller

Comment (knightXun) [inline pkg/controller/job/jobcontroller.go]: yeah, but i have code-OCD.

**Pull request closed without merging**